packages/api/bin/serve.js

```diff
@@ -374,9 +374,9 @@ async function eraseDynamoTables(stackName, systemBucket) {
 
   try {
     await rulesModel.deleteRules();
+    await providerModel.deleteProviders();
     await Promise.all([
```

Comment:
I suggest instead of moving the `await providerModel.deleteProviders()` outside,
```suggestion
    await Promise.allSettled([
```